specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2023-07-01/redisenterprise.json

```diff
@@ -1700,6 +1700,7 @@
       "properties": {
         "id": {
           "type": "string",
+          "format": "arm-id",
```

Comment:
I think we need to also add make changes in "ForceUnlinkParameters" properties, since it also have array of redis databases

Reply:
We should I think add this attribute whenever we use arm-id for databases ARM ID.
"x-ms-arm-id-details": {
              "allowedResources": [
                {
                  "type": "Microsoft.Cache/redisEnterprise/databases"
                }
              ]
            }

Reply:
Added the `x-ms-arm-id-details` here w/ entry  "Microsoft.Cache/redisEnterprise/databases" and also added for `ForceUnlinkParameters`.